.travis.yml

```diff
@@ -6,14 +6,15 @@ services:
   - docker
 
 python:
-  - "3.5"
```

Comment:
Is Python 3.5 not desired anymore? If it is a mistake, you can keep 3.5, 3.6 and 3.7

Reply:
Just read your comment:

>I recommend removing testing on python3.5 and adding python3.7 instead. The tests will then work without any further changes

Well I believe it would be best if we can make it work so it includes 3.5. Is it possible reasonably or is it too complex?